upgrade/breaking-changes/4-0-0.md

```diff
@@ -10,7 +10,66 @@ position: 1
 
 ## Common Changes
 
-## Shape Parameter
+### Icons
+
+#### Icon
+
+The type of the `Icon` parameter changes to `object` from `string` to facilitate the addition of [Telerik Font and Svg Icons]({%slug general-information/font-icons%}).
+
+#### ImageUrl
+
+Removed the `ImageUrl` parameter, use the [Icon](#icon) parameter instead.
```

Comment:
We should explain how to proceed in more detail. The `Icon` parameter does not support `<img>` tags, so this can generate support.